workspaces/cms-scripts/src/utils.ts

```diff
@@ -46,3 +46,44 @@ export async function getFirst<T>(...fns: Array<() => Promise<T>>): Promise<T> {
     cause,
   });
 }
+
+/**
+ * Escape xml function.
+ */
+
+export const escapeXml = (string: string) => {
+  return string.replace(/[<>&'"]/g, (char) => {
+    switch (char) {
+      case '<': return '&lt;';
+      case '>': return '&gt;';
+      case '&': return '&amp;';
+      case '\'': return '&apos;';
+      case '"': return '&quot;';
+      default: return char;
+    }
+  });
+}
+
+/**
+ * Format date function.
+ */
+
+export const formatDate = (date: string) => {
```

Comment:
This is fine, but for future reference we use `date-fns` in most of our projects to handle dates and specifically string formating